packages/ui/package.json

```
@@ -16,8 +17,8 @@
     "@rollup/plugin-replace": "^6.0.0",
     "@rollup/plugin-typescript": "^12.0.0",
     "@types/file-saver": "^2.0.1",
-    "@types/resize-observer-browser": "^0.1.5",
     "@types/node": "^18.0.0",
+    "@types/resize-observer-browser": "^0.1.5",
     "@types/semver": "^7.5.7",
     "autoprefixer": "^10.4.2",
```

We should add deno here so that it gets installed automatically for development:
```suggestion
    "autoprefixer": "^10.4.2",
    "deno": "^2.2.6",
```

---

I have been having trouble after adding this package like `node/20.11.1/bin/node: --watch: not found
error Command failed with exit code 9.` because node_modules/.bin/deno calls Node instead.
Instead I added install Deno instruction in Readme